recipe/conda_build_config.yaml

```diff
@@ -29,3 +22,20 @@ cxx_compiler_version:   # [osx]
 llvm_version:
   - 14.0
   - 15.0
+  - 16.0
+
+# no channel_targets necessary because LLVM rc's not installable without llvm_rc anyway;
+# zip it to avoid using rc-builds for already-released-in-conda-forge LLVM versions
+channel_sources:
+  - conda-forge
+  - conda-forge
+  - conda-forge/label/llvm_rc,conda-forge
```

Comment:
My thinking is: if we _don't_ add the channel_targets, we wouldn't have to rebuild cctools after the rc-phase; the build of cctools against 16.0.0.rc3 would continue to work against 16.0.x GA.

Reply:
OK, I considered this, but thought it would probably work. But fine to add the channel targets.